.github/workflows/cut_release.yaml

```diff
@@ -0,0 +1,27 @@
+name: Create Tag for Release
+
+on:
+  push:
+    branches:
+    - 'master'
+    paths:
+     - 'chart/k8gb/Chart.yaml'
+
+jobs:
+  release:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Get tag
+        id: get_tag
+        run: |
+          new_tag=$(helm show chart chart/k8gb/ 2> /dev/null | awk '/appVersion:/ {print $2}')
```

Comment:
cat chart/k8gb/Chart.yaml to not depend on helm

Reply:
right